backend/.env.sample

```diff
@@ -1 +0,0 @@
-DATABASE="mongodb+srv://<username>:<password>@<clustername>/<database>?retryWrites=true&w=majority" or your local mongo atlas url
```

Comment:
Why you delete this?

Reply:
Rest looks good to me!✌

Reply:
I just renamed the file name to .env and replaced the above mentioned line with my Mongo DB URI. I shouldnt have deleted that actually , I realized it just now.

Reply:
If you want I can create that file again and will not include that file in gitignore.

Reply:
ok ok no problem, I will merge this. Looks good to me.